modules/ocf_mirrors/manifests/init.pp

```diff
@@ -1,32 +1,32 @@
 class ocf_mirrors {
-  include ocf::ssl::default
+  require ocf::ssl::default
```

Comment:
What does this do?

Reply:
https://github.com/ocf/puppet/pull/472#discussion_r244648872